core/src/main/kotlin/com/github/quillraven/darkmatter/screen/MenuScreen.kt

```diff
@@ -1,11 +1,115 @@
 package com.github.quillraven.darkmatter.screen
 
 import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.utils.Align
 import com.github.quillraven.darkmatter.Game
+import com.github.quillraven.darkmatter.asset.I18NBundleAsset
 import com.github.quillraven.darkmatter.asset.MusicAsset
+import com.github.quillraven.darkmatter.ecs.createDarkMatter
+import com.github.quillraven.darkmatter.ecs.createPlayer
+import com.github.quillraven.darkmatter.ecs.system.MoveSystem
+import com.github.quillraven.darkmatter.ecs.system.PlayerAnimationSystem
+import com.github.quillraven.darkmatter.ecs.system.PowerUpSystem
+import com.github.quillraven.darkmatter.ui.SkinImageButton
+import com.github.quillraven.darkmatter.ui.SkinLabel
+import com.github.quillraven.darkmatter.ui.SkinTextButton
+import ktx.actors.onChangeEvent
+import ktx.actors.onClick
+import ktx.ashley.getSystem
+import ktx.scene2d.Scene2dDsl
+import ktx.scene2d.StageWidget
+import ktx.scene2d.actors
+import ktx.scene2d.imageButton
+import ktx.scene2d.label
+import ktx.scene2d.table
+import ktx.scene2d.textButton
+
+private const val PLAYER_SPAWN_Y = 3f
+private const val OFFSET_TITLE_Y = 15f
+private const val MENU_ELEMENT_OFFSET_TITLE_Y = 20f
+private const val MENU_DEFAULT_PADDING = 5f
 
 class MenuScreen(game: Game) : Screen(game, MusicAsset.MENU) {
+    private val bundle = assets[I18NBundleAsset.DEFAULT.descriptor]
+    private val highScoreText = "${bundle["highscore"]}: "
+
+    override fun show() {
+        super.show()
+        engine.run {
+            getSystem<PowerUpSystem>().setProcessing(false)
+            getSystem<MoveSystem>().setProcessing(false)
+            getSystem<PlayerAnimationSystem>().setProcessing(false)
+            createPlayer(assets, spawnY = PLAYER_SPAWN_Y)
+            createDarkMatter()
+        }
+
+        stage.actors {
+            createMenuUI()
+        }
+    }
+
+    private fun @Scene2dDsl StageWidget.createMenuUI() {
+        table {
+            defaults().pad(MENU_DEFAULT_PADDING)
+
+            label(bundle["gameTitle"], SkinLabel.LARGE.name) {
+                wrap = true
+                setAlignment(Align.center)
+            }.cell(expandX = true, fillX = true, padTop = OFFSET_TITLE_Y, padBottom = MENU_ELEMENT_OFFSET_TITLE_Y)
```

Comment:
This is OK, but know that you can also access the cell in the building block. Especially with that many properties it might be cleaner.

```kotlin
label(bundle["gameTitle"], SkinLabel.LARGE.name) { cell ->
  wrap = true
  setAlignment(Align.center)
  cell.apply {
   expandX()
   // Etc.
  }
}
```

It comes down to personal preference, but as a rule of thumb, `cell()` method should be preferred when there is no building block, while the `cell` parameter might be better if there are additional properties set via lambda. E.g.:

```kotlin

label("No building block.").cell(expandX = true)

label("With building block.") { cell ->
  wrap = true
  cell.expandX()
}
```

Reply:
got that! will be changed in the upcoming commit. Since I switched to TextButtons instead of labels I could get rid off all the alignment calls. And I also noticed that I use fillX/expandX for all cells. So, I moved it to the defaults() area.
It will look cleaner. The only cell block left is for the game title and there I used the cell parameter.